poly-commitment/src/tests/commitment.rs

```diff
@@ -141,7 +141,8 @@ fn test_randomised<RNG: Rng + CryptoRng>(mut rng: &mut RNG) {
             for point in eval_points.clone() {
                 chunked_evals.push(
                     poly.to_chunked_polynomial(srs.g.len())
-                        .evaluate_chunks(point),
+                        .evaluate_chunks(point)
+                        .chunks,
```

Comment:
this looks good as a minimal change, but here we should really keep the `ChunkedEvaluations` type and use it as is instead of deconstructing it

Reply:
(this might mean updating EvaluatedCommitment to use that type also)

Reply:
(but perhaps that could be another issue not to grow the scope of this one too much)